pycontext/context.py

```diff
@@ -109,6 +110,25 @@ def __getitem__(self, key):
             raise KeyError(key)
         return value
 
+    def __getattr__(self, key):
+        """
+        Get attribute's value, starting at the current scope and going upward.
+
+        :param key: the name of the attribute
+        :param value: the variable value
+        """
+        if key in object.__getattribute__(self, '__slots__'):
+            super(Context, self).__getattr__(key)
+        else:
+            value, frame = self._find(key)
+            if frame is None:
+                raise AttributeError(
+                    "'{cls}' instance has no attribute '{name}'".format(
```

Comment:
also as a side effect it will bump the coverage back up to 100% 😄 

since `__getattr__` is called only when attribute is not already found unlike `__getattribute__`, its actually impossible to get into branch where attribute being queried is part of slots since `__getattribute__` would return that attribute by that point.